tests/integration/test_vectorizers.py

```diff
@@ -254,7 +265,10 @@ async def aembed_many_func(texts):
 @pytest.mark.asyncio
 async def test_vectorizer_aembed(avectorizer):
     text = "This is a test sentence."
-    embedding = await avectorizer.aembed(text)
+    if isinstance(avectorizer, VoyageAITextVectorizer):
+        embedding = await avectorizer.aembed(text)
+    else:
+        embedding = await avectorizer.aembed(text)
```

Comment:
same question, why do we need this if-else?

Reply:
@justin-cechmanek Same as above, looks like a mistake